contrib/raftexample/main.go

```diff
@@ -33,9 +33,14 @@ func main() {
 	confChangeC := make(chan raftpb.ConfChange)
 	defer close(confChangeC)
 
+	commitC, errorC := make(chan *string), make(chan error)
+	walReady := make(chan struct{})
+	kv := newKVStore(proposeC, commitC, errorC, walReady)
+
 	// raft provides a commit stream for the proposals from the http api
-	commitC, errorC := newRaftNode(*id, strings.Split(*cluster, ","), *join, proposeC, confChangeC)
```

Comment:
why change this interface? `commitC` and `errorC` are scoped by `raftNode`'s lifetime so they shouldn't be passed in as arguments